src/UI/Content/Images/favicon/manifest.json

```diff
@@ -1,53 +1,18 @@
 {
     "name": "Radarr",
     "icons": [
-        {
-            "src": "/Content/Images/favicon/android-chrome-36x36.png",
```

Comment:
Are these not needed anymore?

Reply:
No this is for the older android very early low res phones also android will automatically down res from the high res images